react-task/src/components/Register.js

```diff
@@ -0,0 +1,115 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+const Register = () => {
+  return (
+    <form class="container w-50 d-flex flex-column justify-content-center mt-5">
```

Comment:
HTML semantics is very important, especially when dealing with people that have disabilities. You want the screen reader to read out exactly what is occurring on page if someone is blind and unable to read (https://www.w3schools.com/html/html_accessibility.asp)

In general, a component should start with an `article` element as an article is seen as a component that can be extracted and plugged into any other website.

```suggestion
    <article class="container w-50 d-flex flex-column justify-content-center mt-5">
```